recipes/wk_spectra/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "wk_spectra" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d217783761ae98ef5f2bfc83376d61bcb9d9e59cfbf650e29e21f0a986c996c9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - coverage
+    - matplotlib >=3.0.0
+    - netcdf4 >=1.4.1
+    - nose
+    - numpy >=1.15.2
+    - pandas >=0.23.4
+    - pip
+    - python
+    - scipy >=1.1.0
+    - seaborn ==0.9.0
+    - sphinx
+    - sphinx_rtd_theme
+  run:
+    - coverage
```

Comment:
Are `nose`, `coverage`, `sphinx` and `sphinx_rtd_theme` actually used at run time? If not, you can remove them from the `run` section.